client/src/containers/FormBuilderLoader/FormBuilderLoader.js

```diff
@@ -339,7 +352,7 @@ function mapStateToProps(state, ownProps) {
   const reduxFormState =
     state.form &&
     state.form.formState &&
-    getIn(state.form.formState, ownProps.identifier);
+    getIn(state.form.formState, createIdentifier(ownProps));
```

Comment:
could even reassign the identifier value here so `const identifier = createIdentifier(ownProps);`

it does have the cross-wiring, so I'm a little adverse to that idea, but it's possible and it wouldn't stack/recurse